meson_options.txt

```diff
@@ -1,3 +1,4 @@
 option('examples', type: 'boolean', value: false, description: 'Build example applications')
 option('docs', type: 'boolean', value: false, description: 'Build devhelp API documentation')
 option('tests', type: 'boolean', value: false, description: 'Build tests')
+option('introspection', type: 'boolean', value: false, description: 'Build gobject-introspection data')
```

Comment:
Setting to false by default will break packages. Adding the option is fine but I don't see why the default behavior needs to change.

Reply:
You're right, that makes sense. I have just pushed an update to have the option enabled by default 👍🏼